ui/Screen/Lock.svelte

```diff
@@ -43,11 +43,12 @@
 
 <div class="lock">
   <Emoji emoji={'🚪'} size="huge" style="margin-bottom: 0.75rem;" />
-  <h3>Unlock the app</h3>
+  <h3>Knock, knock</h3>
+  <p>Enter your passphrase to unlock your Radicle secret key.</p>
   <div class="form">
     <Input.Password
       autofocus
-      placeholder="Enter your passphrase"
+      placeholder=""
```

Comment:
Why remove the placeholder? It feels like bad UX to me.

Reply:
It's written right above the field, so it's redundant.

Reply:
Agree

Reply:
I don't think we have any fields without placeholders. It's a must.

Reply:
```suggestion
      placeholder="Enter your passphrase"
```

oki, changing it back.